### PR TITLE
Add wolfi-base

### PR DESCRIPTION
**Pull request by jdolitsky**

### CODE REVIEW

**Review by ajayk — APPROVED** [review]

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -77,3 +79,27 @@ jobs:
...
+        if: matrix.imageName == 'wolfi-base'
+        env:
+          TF_VAR_target_repository: docker.io/${{ env.WOLFIOS_DOCKERHUB_USER }}/${{ matrix.imageName }}
+        run: |
+          set -x
+          cd images/${{ matrix.imageName }}/
+          # Remove any terraform state from the push to GHCR
+          rm -rf .terraform* terraform*
+          terraform init
+          terraform apply -auto-approve
```
Comment (imjasonh) [inline .github/workflows/release.yaml:105]:
```suggestion
```

I don't think we plan to push to Dockerhub yet.

File: README.md [hunk 2]
```diff
@@ -40,6 +40,21 @@ The [melange](./images/melange) image contains
...
 
+### wolfi-base
+
+The [wolfi-base](./images/wolfi-base) image contains
+the `wolfi-base` package (which provides `busybox` and `apk`).
+
+on Docker Hub:
+```
+wolfios/wolfi-base:latest
+```
```
Comment (imjasonh) [inline README.md:51]:
```suggestion
```

File: .github/workflows/release.yaml (hunk 1 above)
Comment (kaniini) [reply .github/workflows/release.yaml:105]: I mean, we can.  The RAPID just says it is not an immediate requirement, but if it is already done -- why not?

**Comment by imjasonh** [comment]
Opened https://github.com/wolfi-dev/tools/pull/47 to move on this while Josh is out.

**Comment by xnox** [comment]
We have https://hub.docker.com/r/chainguard/wolfi-base now so i think this is obsolete.

**Pull request closed without merging**